repo/open.go

```diff
@@ -34,9 +34,9 @@ const CacheDirMarkerFile = "CACHEDIR.TAG"
 // CacheDirMarkerHeader is the header signature for cache dir marker files.
 const CacheDirMarkerHeader = "Signature: 8a477f597d28d172789f06886806bc55"
 
-// defaultRepositoryBlobCacheDuration is the duration for which we treat cached kopia.repository
+// DefaultRepositoryBlobCacheDuration is the duration for which we treat cached kopia.repository
 // as valid.
-const defaultRepositoryBlobCacheDuration = 15 * time.Minute
+const DefaultRepositoryBlobCacheDuration = 15 * time.Minute
```

Comment:
Wondering if this change to export this const is actually needed. Seems to only be consumed by repo tests, but may be more correct to pull it out of the test environment repository directly?

Reply:
You are right. Actually, this PR is a spin off from the whole thing here:

https://github.com/kopia/kopia/pull/1622

And there we use the const outside of the module. But you are right, lets expose the const when we get there.